strawberry/experimental/pydantic/object_type.py

```diff
@@ -67,18 +67,24 @@ def replace_pydantic_types(type_: Any):
         return replaced_type
 
     if issubclass(type_, BaseModel):
-        if hasattr(type_, "_strawberry_type"):
-            return type_._strawberry_type
+        if is_input:
+            if hasattr(type_, "_strawberry_input_type"):
+                return type_._strawberry_input_type
+            else:
+                raise UnregisteredTypeException(type_)
         else:
-            raise UnregisteredTypeException(type_)
+            if hasattr(type_, "_strawberry_type"):
+                return type_._strawberry_type
+            else:
+                raise UnregisteredTypeException(type_)
```

Comment:
nit; can we do something like: `attr = "_strawberry_input_type" if is_input else "_strawberry_type"`?

Reply:
Good call